fem/bilininteg.hpp

```diff
@@ -2145,24 +2100,34 @@ class VectorMassIntegrator: public BilinearFormIntegrator
    const GeometricFactors *geom;  ///< Not owned
    int dim, ne, nq, dofs1D, quad1D;
 
+   // CEED extension
+   CeedData* ceedDataPtr;
+
 public:
    /// Construct an integrator with coefficient 1.0
    VectorMassIntegrator()
-      : vdim(-1), Q_order(0), Q(NULL), VQ(NULL), MQ(NULL) { }
+      : vdim(-1), Q_order(0), Q(NULL), VQ(NULL), MQ(NULL), ceedDataPtr(NULL) { }
    /** Construct an integrator with scalar coefficient q.  If possible, save
        memory by using a scalar integrator since the resulting matrix is block
        diagonal with the same diagonal block repeated. */
    VectorMassIntegrator(Coefficient &q, int qo = 0)
-      : vdim(-1), Q(&q) { VQ = NULL; MQ = NULL; Q_order = qo; }
+      : vdim(-1), Q(&q), VQ(NULL), MQ(NULL), Q_order(qo), ceedDataPtr(NULL) { }
    VectorMassIntegrator(Coefficient &q, const IntegrationRule *ir)
-      : BilinearFormIntegrator(ir), vdim(-1), Q(&q)
-   { VQ = NULL; MQ = NULL; Q_order = 0; }
+      : BilinearFormIntegrator(ir), vdim(-1), Q(&q), VQ(NULL), MQ(NULL),
+        Q_order(0), ceedDataPtr(NULL) { }
```

Comment:
Can you put `Q_order(0)` before `Q(&q)` here? The compiler will complain about initialization order otherwise (`nvcc` doesn't complain, but with `MFEM_USE_CUDA=NO` you should see it).

Reply:
done.